pkg/host/internal/lib/netlink/netlink.go

```diff
@@ -47,6 +47,14 @@ type NetlinkLib interface {
 	// VDPANewDev adds new VDPA device
 	// Equivalent to: `vdpa dev add name <name> mgmtdev <mgmtBus>/mgmtName [params]`
 	VDPANewDev(name, mgmtBus, mgmtName string, params netlink.VDPANewDevParams) error
+	// DevlinkGetDeviceParamByName returns specific parameter for devlink device
+	// Equivalent to: `devlink dev param show <bus>/<device> name <param>`
+	DevlinkGetDeviceParamByName(bus string, device string, param string) (*netlink.DevlinkParam, error)
```

Comment:
do we really need `ByName` suffix?

Reply:
This is a wrapper for the function  `DevlinkGetDeviceParamByName` in the netlink library. I think we should use exact same name as original (wrapped) function.